spec/support/issue.cr

```diff
@@ -0,0 +1,23 @@
+class Issue < BaseModel
+  COLUMN_SQL = "issues.id, issues.status, issues.role"
+
+  avram_enum Status do
+    Opened = 0
```

Comment:
Do you have to define these values? It looks like it just proxies to the crystal `enum`, so maybe we can leave them off here?

Reply:
Yes, it is just proxies to enum, so we could leave them off here. Can remove them later :)

Reply:
Fixed this. So resolving it :)